plugins/@grouparoo/snowflake/__tests__/table-import/importProperty.ts

```diff
@@ -67,7 +77,7 @@ describe("snowflake/table/recordProperty", () => {
     record = await helper.factories.record();
     await record.addOrUpdateProperties({
       userId: [1],
-      email: ["ejervois0@example.com"],
+      email: ["ejervoiS0@example.com"],
```

Comment:
In the CSV sample data above, the email is `ejervois0@example.com` (lowercase) - how did it get uppercased?

Reply:
I wasn't sure how to get the csv to sync, I edited the demo data directly in snowflake. Could use a hand!

Reply:
I think it was rather manual by uploading the CSV.  The CSV data and what's loaded into snowflake should match.

Reply:
Then this should have parity now I think.